packages/react-scripts/config/paths.js

```diff
@@ -78,9 +78,12 @@ module.exports = {
   dotenv: resolveApp('.env'),
   appPath: resolveApp('.'),
   appBuild: resolveApp('build'),
+  devAppBuild: resolveApp('dev'),
   appPublic: resolveApp('public'),
-  appHtml: resolveApp('public/index.html'),
```

Comment:
Follow chrome extension terminology of calling the [extension UI a popup](https://developer.chrome.com/extensions/user_interface#popup)